test/unit/composite-view/composite-view.spec.js

```diff
@@ -370,7 +369,7 @@ describe('composite view', function() {
       expect(Backbone.Marionette.Renderer.render.callCount).to.equal(2);
     });
 
-    it('should destroy all of the child collection child views', function() {
+    it.skip('should destroy all of the child collection child views', function() {
```

Comment:
I believe this is an implementation detail of `NextCollectionView` always calling the `_destroyChildren` method inside render